halium-install

```diff
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/bin/bash
```

Comment:
Depends what's symlinked to /bin/sh. On Arch it's bash by default but I don't know about other distros. /bin/bash ensures we get bash.